README.md

```diff
@@ -53,7 +52,7 @@ Consult the [LORIS Wiki](https://github.com/aces/Loris/wiki/Setup) page on this
 
     <i>$projectname ⇾ "loris" or one-word project name</i>
 
-2. Get code:
+2. Get the code:
```

Comment:
Comment for line 40-44

How can you use `sudo` if you're not yet in the sudo group?
Plus the sudo group doesn't exist on CentOS7.

should be something like
As `root` user
```
useradd -b -m -U -s /bin/bash lorisadmin
passwd lorisadmin
```
give root permissions to the `lorisadmin` user:
`echo  'lorisadmin    ALL=(ALL)       ALL' >> /etc/sudoers`

then switch to the lorisadmin user:
`su - lorisadmin`

Reply:
I don't think `lorisadmin` should have sudo or needs it.

Instead the instructions should have `su - lorisadmin` as the last command to run and the sentence 

> <b>Important ⇾ All steps from this point forward must be executed by lorisadmin user</b>

Should be moved below that command.

Reply:
@johnsaigle sorry, should have removed this as this file is more Ubuntu targeted.

Reply:
@PapillonMcGill @johnsaigle I think it is better to add lorisadmin user in the root group, just like the wiki guide for ubuntu.https://github.com/aces/Loris/wiki/Installing-Loris-in-Brief#creating-a-user-for-loris

Reply:
@kongtiaowang Those instructions are the equivalents of the link you pointed for CentOS. If you try the instructions as directed in https://github.com/aces/Loris/wiki/Installing-Loris-in-Brief#creating-a-user-for-loris on CentOS, the result will not work.
Note that these change are for the READMECentOS.md file.